.travis.yml

```diff
@@ -1,7 +1,26 @@
 language: node_js
 
 node_js:
-  - 4.8.6
+  - node
+
+branches:
+  except:
+  - /^v?\d+\.\d+\.\d+$/
 
 before_install:
- - yarn config set ignore-engines true
\ No newline at end of file
+  # If package.json major and minor versions match last tag, then increment last tag. Else use package.json major.minor.0.
+  - "{ sed -nE 's/^[ \\t]*\"version\": \"([0-9]{1,}\\.[0-9]{1,}\\.)[0-9x]{1,}\",$/\\1/p' package.json; git describe --abbrev=0 | sed -E 's/^v([0-9]{1,}\\.[0-9]{1,}\\.)([0-9]{1,})$/\\1 \\2/g'; } | tr \"\\n\" \" \" | awk '{printf($1==$2?\"v\"$2$3+1:\"v\"$1\"0\")}' | xargs -I {} git tag -a {} -m \"{}\"\n"
+  - npm --no-git-tag-version version from-git
+  - yarn config set ignore-engines true
+
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script: git push --tags
+    on:
+      branch: master
+  - provider: npm
+    email: "paul.heasley@gmail.com"
```

Comment:
just a quesiton, should we use the pageup account that created by Karthik?

Reply:
As far as I'm aware there is no pageup account. There's a pageup organisation that Karthik and I are members of. This is a smell I agree but I honestly don't know another way of doing it.

Reply:
I'll just try removing the email and see if it's optional

Reply:
no problem,  the reason I ask is that day we saw the pageup is having Karthik's profile photo on it, so I thought there is a pageup specific account.